businesscentral-monitoring/modules/businesscentral-monitoring/module.yaml

```diff
@@ -25,8 +25,8 @@ ports:
 - value: 8001
 artifacts:
 - name: "rhpam_business_central_monitoring_distribution.zip"
-  # rhpam-7.12.0-monitoring-ee7.zip
-  md5: "71fdacfb572bdb1699ec42880158fbe6"
+  # rhpam-7.12.1-monitoring-ee7.zip
```

Comment:
are `RHPAM_BUSINESS_CENTRAL_MONITORING_VERSION` and `PRODUCT_VERSION` ok? shouldn't they be `7.12.1`?